package.json

```diff
@@ -88,13 +88,16 @@
     "@progress/kendo-angular-pdf-export": "^1.0.1",
     "@progress/kendo-drawing": "^1.4.2",
     "angular2-text-mask": "^8.0.4",
+    "body-parser": "^1.19.0",
     "bootstrap": "^3.3.7",
     "bootstrap-loader": "^2.2.0",
     "bootstrap-sass": "^3.3.7",
     "classlist.js": "^1.1.20150312",
     "core-js": "^2.4.1",
+    "cors": "^2.8.5",
     "crypto-js": "^3.1.9-1",
     "ecstatic": "^2.0.0",
+    "express": "^4.16.4",
```

Comment:
I'm not sure where this is coming from. but we shouldn't add these dependencies to package.json.